kie-server-parent/kie-server-remote/kie-server-client/src/test/java/org/kie/server/client/LoadBalancerClientTest.java

```diff
@@ -290,6 +293,232 @@ public void testDefaultLoadBalancerNotValidHost() throws Exception {
         assertEquals("Server version", "1", response.getResult().getVersion());
     }
 
+    @Test
+    public void testMultipleConcurrentFailRequestsForLoadBalancerWithSingleServer() throws Exception {
+
+        class SendKieRequestThread extends Thread {
+            CountDownLatch startLatch;
+            CountDownLatch stopLatch;
+            int threadNo;
+            KieServicesClient kieClient;
+
+            SendKieRequestThread(int threadNo, CountDownLatch startLatch, CountDownLatch stopLatch, KieServicesClient client) {
+                this.startLatch = startLatch;
+                this.stopLatch = stopLatch;
+                this.threadNo = threadNo;
+                this.kieClient = client;
+            }
+
+            @Override
+            public void run() {
+                try {
+                    startLatch.await();
+                    System.out.println("Th#" + threadNo + " Calling Kie Server ");
```

Comment:
could you please replace system out with logger?

Reply:
logger is updated